src/status/libstatus/wallet.nim

```diff
@@ -32,6 +33,36 @@ proc getWalletAccounts*(): seq[WalletAccount] =
     let msg = getCurrentExceptionMsg()
     error "Failed getting wallet accounts", msg
 
+proc getTransfersByAddress*(address: string): seq[Transaction] =
+  try:
+    var response = status.getBlockByNumber("latest")
+    let latestBlock = parseJson(response)["result"]
+    
+    var transactionsResponse = status.getTransfersByAddress(address, latestBlock["number"].getStr, "0x14")
```

Comment:
The `limit` parameter here for `getTransfersByAddress` is hardcoded to `20`.

This needs to change in future commits once we know how we want to handle many transactions

Reply:
```suggestion
    let transactionsResponse = status.getTransfersByAddress(address, latestBlock["number"].getStr, "0x14")
```